spec/lib/vmdb/gettext/debug_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe Vmdb::Gettext::Debug do
+  include Vmdb::Gettext::Debug
```

Comment:
> This line bothers me as it could lead to test contamination / sporadic bugs, but I'm not sure if including here includes it only in the example group or if it's global. If it's being included globally, we have to find another way to test

Good question, I don't know... rspec magic :sparkles:... 

Either way, I agree, it's not a good idea to include modules outside of a class/module.
At the very least, I think we can tag the examples that need it and include them in the spec_helper like we do for other specs.